# Federated Learning for the pathogenicity annotation of genetic variants in multi-site clinical settings [![DOI](https://zenodo.org/badge/950719898.svg)](https://doi.org/10.5281/zenodo.16029049)

available at [medrXiv](https://www.medrxiv.org/content/10.1101/2025.04.03.25325184v1)

## Abstract
Rare diseases collectively affect 5% of the population. However, fewer than 50% of rare disease patients receive a molecular diagnosis after whole genome sequencing. Supervised machine Learning is a valuable approach for the pathogenicity scoring of human genetic variants. However, existing methods are often trained on curated but limited central repositories, resulting in poor accuracy when tested on external cohorts. Yet, large collections of variants generated at hospitals and research institutions remain inaccessible to machine-learning purposes because of privacy and legal constraints. Federated learning (FL) algorithms have been recently developed enabling institutions to collaboratively train models without sharing their local datasets. Here, we present a proof-of-concept study evaluating the effectiveness of federated learning for the clinical classification of genetic variants. A comprehensive array of diverse FL strategies was assessed for coding and non-coding Single Nucleotide Variants as well as Copy Number Variants. Our results showed that federated models generally achieved comparable or superior performance to traditional centralized learning. In addition, federated models reached a robust generalization to independent sets with smaller data fractions as compared to their centralized model counterparts. Our findings support the adoption of FL to establish secure multi-institutional collaborations in human variant interpretation.

### Authors
Nigreisy Montalvo, Francisco Requena, Emidio Capriotti & Antonio Rausell

Correspondence to: antonio.rausell@institutimagine.org

